pkg/apis/velero/v1/backupstoragelocation_types.go

```diff
@@ -74,6 +74,10 @@ type BackupStorageLocationStatus struct {
 	// +nullable
 	LastValidationTime *metav1.Time `json:"lastValidationTime,omitempty"`
 
+	// Message is a message about the backup storage location's status.
+	// +optional
+	Message string `json:"message,omitempty"`
```

Comment:
this is actually "error message" right?  
If that's the case, the key and comment should be updated.